service/src/main/kotlin/io/provenance/digitalcurrency/consortium/stream/EventStreamConsumer.kt

```diff
@@ -145,10 +176,14 @@ class EventStreamConsumer(
         val filteredTransfers = transfers.filter { it.fromAddress.isNotEmpty() && it.toAddress.isNotEmpty() }
             .filter { it.denom == bankClientProperties.denom }
             .mapNotNull { event ->
-                log.debug("MarkerTransfer - tx: ${event.txHash} from: ${event.fromAddress} to: ${event.toAddress} amount: ${event.amount} denom: ${event.denom}")
+                log.info("MarkerTransfer - tx: ${event.txHash} from: ${event.fromAddress} to: ${event.toAddress} amount: ${event.amount} denom: ${event.denom}")
```

Comment:
keeping all these these the same or reverting for release?

Reply:
I can move back to debug.